roles/online_docs/templates/mkdocs/docs/logins.md

```diff
@@ -153,7 +153,7 @@ The following assumes:
         #
         Host {% for jumphost in groups['jumphost'] %}{{ jumphost | regex_replace('^' + ai_jumphost + '\\+','')}}80+* {% endfor %}{% raw %}{% endraw %}
             PasswordAuthentication No
-            ProxyCommand ssh -X -q youraccount@$(echo %h | sed 's/+[^+]*$//'){% if slurm_cluster_domain | length %}.{{ slurm_cluster_domain }}{% endif %} -W $(echo %h | sed 's/^[^+]*+//'):%p -p 80
+            ProxyCommand ssh -X -q youraccount@$(echo %h | sed 's/80+[^+]*$//'){% if slurm_cluster_domain | length %}.{{ slurm_cluster_domain }}{% endif %} -W $(echo %h | sed 's/^[^+]*+//'):%p -p 80
```

Comment:
Could you please explain what this does, i am unable to review this.

Reply:
the ProxyCommand directives enable
```
ssh jumphost+destination
```
as documented. We use this syntax also for the existing/old clusters. For SSH over port 80 we use:
```
ssh jumphost80+destination
```
The first combination of ```echo``` and ```sed``` selects the name of the jumphost (optional with FQDN when the jumphost has an DNS entry) and the second combination selects the destination, so it becomes:
```
ProxyCommand ssh -X -q youraccount@jumphost.some.domain -W destination:%p -p 80
```